### DISCUSSION TITLE
Mineral Scanner on navigation terminal (it's a problem)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#9822] [Unstable feedback]
You can buy the Mineral Scanner on a tier 1 Scout/Transport submarine, sometimes with starter money:

![image](https://user-images.githubusercontent.com/73229309/188268979-0e24a12a-0082-4dac-b566-0e3d764946b5.png)

1. That's a powerful upgrade, from the very start of the game. Finding resources becomes significantly easier as a result: you simply can't miss minerals.
2. You don't need a separate "sonar operator" any more, the captain can do the scanner so there are less things to do on the ship for the crew
3. Resource hoarding players (there are a LOT of them) will try and harvest every single mineral they see. This will significantly slow down rounds and increase the amount of times you spend in the early biomes.

To avoid that:

**Increase the cost (or reduce the accessibility) of the mineral scanner**. You shouldn't be able to get it as early as cold caverns. Maybe restrict it to Tier2+ submarines.

**Merge Directional Ping and Mineral Scanner so the captain can't drive and scan at the same time.**
Directional sonar has always been useless + this brings back the "decicated" sonar operator role and also makes scanning take a little more skill. 

![image](https://user-images.githubusercontent.com/73229309/188269783-53fbea7d-dcf7-491b-b10d-1a4fa3996bde.png)

**Comment by gerakis** [comment]
I dont know about this one. Hardly any ships have scanners to begin with, and even then nobody ever uses them. Usually once a ship has a good set of resources people stop mining. It will definitely slow down the beginning levels though.

**Comment by mLuby** [comment]
> separate "sonar operator"

I've never seen a separate sonar operator in the game. (I'd love to see it once [sonar gameplay evolves](https://github.com/Regalis11/Barotrauma/discussions/5219).) Anyway only R-29 and Berelia even _have_ Sonar Monitors.

> Resource hoarding players…increase times you spend in the early biomes

Alternative is to make minerals more scarce early on. It would make sense that the most inhabited biomes are picked clean while later wilder biomes are rich in unclaimed natural resources.

**Comment by Ek-Videogames** [comment]
> Resource hoarding players…increase times you spend in the early biomes

Well you patched out every other way of getting resources what do you expect? 
That & even more elaborate forms of outpost theft because all the shop owners are scam artists.

Mining isn't the _worst_ mechanic in the game, but its up there.
Wreck looting was so popular because it was more interesting gameplay & lucrative enough to be worth the timesink. 
Mining kind of accomplishes neither well, it just causes the round length to drag on considerably while scouring the map walls & praying to RNG that the game gives you one of the minerals that deconstructs into useful resources.

So whatever it is we decided we want the player spending 90% of their playtime doing, _(Be it mining, exploring, trading, fabricating)_ it should be examined as to weather or not it's actually engaging with it in a meaningful way.

**Reply by gerakis** [reply]
I agree with this. I rarely have a fun time mining. Maybe replace it with something else, like just make wrecks more common and more lucrative. Or more pirates. Nobody ever complains about pirate missions.

**Reply by LeakingAmps** [reply]
I miss there being tons of wrecks in past playthoughs. I had fun in them.

**Comment by Likvidus** [comment]
This will be a good change. It is high time to give meaning to the use of directional sonar.

**Comment by Ek-Videogames** [comment]
Also you realize if mineral scanner is removed the alternative is people will just go back to having somebody sitting outside the sub with a handheld sonar looking for minerals. 

So I mean while the points make sense, people would just cheese it anyways.
